examples/fluent-bit/grafana-loki/README.md

```diff
@@ -0,0 +1,23 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit and Firelens](https://grafana.com/docs/loki/latest/clients/fluentbit/). The Loki data source provides access to Loki, Grafana’s log aggregation system.
```

Comment:
>,  see: [Fluent Bit and Firelens]
Change to something like: , see [Fluent Bit and FireLens in the documentation Grafana]

Reply:
Sure made the change